lib/routes/emails.js

```diff
@@ -31,16 +30,11 @@ module.exports = (log, db, mailer, config, customs, push) => {
         log.begin('Account.RecoveryEmailEnabled', request)
```

Comment:
I wonder if this entire endpoint can be safely removed, the content server will not allow a user to view `/settings` if their session is unverified.

Reply:
Removing the endpoint will also require a content-server update.

Reply:
> I wonder if this entire endpoint can be safely remove

I think we might be able to reuse https://github.com/mozilla/fxa-auth-server/blob/master/lib/routes/session.js#L68 and attach the `sessionToken.tokenVerified` to response. Then we would be able to remove this endpoint. In either of those cases, it would require updates to content server and need to be done in stages. Maybe new issue to track that?

> the content server will not allow a user to view /settings if their session is unverified.

Session unverified in this sense means that the session can not be used to retrieve sync keys.